ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/validator/preexecution/RawKeyValuePairsWithLogEntry.scala

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.ledger.validator.preexecution
+
+import com.daml.ledger.participant.state.kvutils.Bytes
+import com.daml.ledger.validator.SubmissionValidator.RawKeyValuePairs
+
+/**
+  * A key value pair with the log entry.
+  */
+case class RawKeyValuePairsWithLogEntry(logEntry: (Bytes, Bytes), state: RawKeyValuePairs)
```

Comment:
I have two minor suggestions for improving readability:
* Put the entries in the same order as the name suggests.
* Expand the pair with explicit field names (e.g., `logEntryKey`, `logEntryValue`).

Reply:
Done.